### ISSUE TITLE
Jcarousel FireFox Issue

### ISSUE THREAD

**Issue by borconi** [jsor/jcarousel#145]
Hi. 
I have a strange problem, I try to create a Jcarousel with a PHP script, depending on the server response. It works fine in IE, Chrome, but in FireFox I get one endless loop. I read that height and width can be a problem, but there are already set. Here is my HTML code generated by the PHP:
&lt;!DOCTYPE HTML&gt;

&lt;html&gt;

&lt;script type=&quot;text/javascript&quot; src=&quot;http://ajax.googleapis.com/ajax/libs/jquery/1.5.0/jquery.min.js&quot;&gt;&lt;/script&gt;

&lt;script TYPE=&quot;text/javascript&quot; SRC=&quot;../js/jquery.jcarousel.min.js&quot;&gt;&lt;/script&gt;

&lt;script type=&quot;text/javascript&quot; src=&quot;../js/jquery.alerts.js&quot;&gt;&lt;/script&gt;

&lt;link rel=&quot;stylesheet&quot; type=&quot;text/css&quot; media=&quot;all&quot; href=&quot;../css/jcarousel.css&quot; /&gt;

&lt;link rel=&quot;stylesheet&quot; href=&quot;../css/jquery.alerts.css&quot;&gt; 

  &lt;style type=&quot;text/css&quot;&gt; 

.jcarousel-skin-tango .jcarousel-container {

```
 background: none;
```

 }

.jcarousel-skin-tango .jcarousel-item {

```
width: 170px;

height: 170px;

margin-left:9px;
```

}

.jcarousel-skin-tango .jcarousel-clip-horizontal {

```
height: 170px;
```

}

.jcarousel-skin-tango .jcarousel-container-horizontal {

```
width: 245px;

padding: 0px 40px;
```

}

/**
- Overwrite for having a carousel with dynamic width.
  
  */

.jcarousel-skin-tango .jcarousel-container-horizontal {

```
width: 95%;
```

}

.jcarousel-skin-tango .jcarousel-clip-horizontal {

```
width: 100%;
```

}
# demo3 .box2 {

  display:list-item;

  width: 175px;

  height:155px;

  padding-left: 0px;

  padding-bottom: 5px;

  margin-right: 0px;

  margin-bottom: 10px;

  border-style:solid;

  border-width: thin;

  border-color:#504C41;

  -moz-border-radius: 10px;

  border-radius: 10px;

  -moz-border-top-left-radius: 0px;

  border-top-left-radius: 0px;

  }

  &lt;/style&gt; 

&lt;script&gt; 

var sbx = window.parent.Shadowbox; 

function openTopSBX(el)

{ if(sbx){ 

sbx.open( { 

```
    content : el.href , 

    player : 'iframe' , 

    title : el.title||'' ,

    width:900,

    height:500 

     } ); 
```

return false; }

else{ return true; } 

} 

&lt;/script&gt;  

&lt;body style=&quot;background-color:transparent&quot;&gt;

&lt;div STYLE='position:absolute;top:5px;left:5%;'&gt;&lt;font COLOR=#646A60 SIZE=5em&gt;&lt;b&gt;+P - Parkol&Atilde;&sup3; &lt;/b&gt;&lt;/font&gt;&lt;/div&gt;

&lt;div STYLE='position:absolute;top:5px;right:5%'&gt;&lt;span STYLE='position:relative;right:42px;'&gt;&lt;font COLOR=#646A60 SIZE=4em&gt;TAL&Atilde;�LATOK SZ&Atilde;�MA: &lt;/span&gt;

&lt;div STYLE='color:white;background-color:#646A60;  -moz-border-radius: 10px;  border-radius: 10px; position:absolute;right:0px;top:0px;width:30px'&gt;&lt;b&gt;&lt;font COLOR=white&gt;&lt;center&gt;

6&lt;/b&gt;&lt;/center&gt;&lt;/font&gt;&lt;/div&gt;&lt;/DIV&gt;&lt;br&gt;&lt;br&gt;

&lt;div id=demo3&gt;

&lt;ul id='mycarousel' class='jcarousel-skin-tango'&gt;

&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=1 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=1'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;Opel Corsa 5 ajto &lt;br&gt; Enjoy 1.4 (Benzin) &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;3 113 940 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=1 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=14 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=14'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;Opel test1 &lt;br&gt;  &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;0 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=14 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=15 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=15'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;opel test2 &lt;br&gt;  &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;0 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=15 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=16 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=16'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;opel test3 &lt;br&gt;  &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;0 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=16 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=17 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=17'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;opel test4 &lt;br&gt;  &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;0 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=17 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;li&gt;

 &lt;div class='box2' STYLE='width: 175px;  height:155px;'&gt;

 &lt;img SRC='../images/icons/p.png' STYLE='top:4px;left:4px; cursor:pointer; float:left;' BORDER=0 ertek=2 CLASS=parkolo&gt;

 &lt;center&gt;&lt;img SRC='kiskep.php?id=2'&gt;&lt;/img&gt;&lt;/center&gt;

 &lt;font color=#646965&gt;&lt;center&gt;Opel Zafira &lt;br&gt; Enjoy &lt;/center&gt;&lt;/font&gt;

 &lt;div STYLE='background-color:#646A60;width:100%;margin-top:4px;'&gt;&lt;center&gt;&lt;Font COLOR=#F7C833&gt;3 450 000 &lt;/b&gt; Ft &lt;/center&gt;&lt;/font&gt;

&lt;/div&gt;

&lt;center&gt;&lt;a HREF=leiras.php?id=2 TITLE='R&Atilde;&copy;szletes leir&Atilde;&iexcl;s' rel='nofollow' target='_top' onclick='return openTopSBX(this);'&gt;r&Atilde;&copy;szletek&lt;/a&gt;&lt;/center&gt;

&lt;/div&gt;

&lt;/li&gt;&lt;/ul&gt;&lt;/div&gt;

&lt;script&gt;

$('#mycarousel').jcarousel({

```
    itemFallbackDimension:175,      

    visible: 5,

    scroll:1});
```

&lt;/script&gt;&lt;script&gt;

$('.parkolo').click(function()

{

$.ajax({

```
  url: 'mentes.php',

  dataType: &quot;html&quot;,

  data: ({adatok: $(this).attr('ertek')}),

  type: &quot;POST&quot;,

  success: function(strData) { jAlert(strData, 'Ment&Atilde;&copy;s');  },

  error: function() {alert('hiba');}
```

});

});

&lt;/script&gt;
